client/src/app/site/pages/organization/pages/accounts/pages/account-list/components/account-merge-dialog/account-merge-dialog.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, Inject } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { ViewUser } from 'src/app/site/pages/meetings/view-models/view-user';
+
+export type AccountMergeDialogData = { choices: ViewUser[] };
+export type AccountMergeDialogAnswer = number | null;
+
+@Component({
+    selector: `os-account-merge-dialog`,
+    templateUrl: `./account-merge-dialog.component.html`,
+    styleUrls: [`./account-merge-dialog.component.scss`]
+})
+export class AccountMergeDialogComponent {
+    public constructor(
+        public dialogRef: MatDialogRef<AccountMergeDialogComponent, AccountMergeDialogAnswer>,
+        @Inject(MAT_DIALOG_DATA) public data: AccountMergeDialogData
+    ) {}
+
+    public selectedUserId: number;
+
+    public get sortedUsers(): ViewUser[] {
+        return this.data.choices.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
```suggestion
    ) {
        return this.data.choices.sort((a, b) => a.name.localeCompare(b.name));
    }

    public selectedUserId: number;

    public get sortedUsers(): ViewUser[] {
        return this.data.choices;
```

Maybe also rename. No need to resort the choices on every change detection cycle.

Reply:
done